lib/benchee/formatters/csv.ex

```diff
@@ -52,28 +50,21 @@ defmodule Benchee.Formatters.CSV do
       ...> 				sample_size:   8
       ...> 			}
       ...> 		}
-      ...> 	],
-      ...> 	configuration: %Benchee.Configuration{
-      ...> 		formatter_options: %{csv: %{file: "my_file.csv"}}
-      ...> 	}
+      ...> 	]
       ...> }
       iex> suite
-      iex> |> Benchee.Formatters.CSV.format
-      iex> |> Enum.fetch!(0)
-      iex> |> (fn({rows, _}) -> Enum.take(rows, 2) end).()
+      iex> |> Benchee.Formatters.CSV.format(%{})
+      iex> |> elem(0)
+      iex> |> (fn rows -> Enum.take(rows, 2) end).()
       ["Name,Input,Iterations per Second,Average,Standard Deviation,Standard Deviation Iterations Per Second,Standard Deviation Ratio,Median,Minimum,Maximum,Sample Size\\r\\n",
        "My Job,Some Input,2.0e3,500.0,200.0,800.0,0.4,450.0,200,900,8\\r\\n"]
 
   """
-  @spec format(Suite.t) :: [{Enumerable.t, String.t}]
-  def format(%Suite{scenarios: scenarios, configuration: configuration}) do
-    sorted_scenarios = Enum.sort_by(scenarios, fn(scenario) -> scenario.input_name end)
-    filename = get_filename(configuration)
+  @spec format(Suite.t(), map) :: {Enumerable.t(), Enumerable.t()}
+  def format(%Suite{scenarios: scenarios}, _) do
```

Comment:
I think it'd be ok to introduce a default here - what do you think?

Reply:
eh I mean some meaningless default so people don't have to call format with `nil` as the last argument to satisfy the 2 arguments interface

Reply:
Personally I think it makes more sense just to stick with the behaviour we’re implementing. We only ever directly call this in tests.

Reply:
Hm, true - people other than ourselves calling this are probably mostly 0. Convinced :grin: